internal/pkg/argocd/argocd.go

```diff
@@ -147,8 +151,16 @@ func generateArgocdAppDiff(ctx context.Context, keepDiffData bool, app *argoappv
 
 // Should return output that is compatible with github markdown diff highlighting format
```

Comment:
i know it's from before but it would be great if we could make the comment `godoc` compatible.
```
// diffLiveVsTargetObject return output that is compatible with github markdown diff highlighting format
```